core/string/translation_server.cpp

```diff
@@ -578,7 +574,7 @@ void TranslationServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_tool_locale"), &TranslationServer::get_tool_locale);
 
 	ClassDB::bind_method(D_METHOD("compare_locales", "locale_a", "locale_b"), &TranslationServer::compare_locales);
-	ClassDB::bind_method(D_METHOD("standardize_locale", "locale"), &TranslationServer::standardize_locale);
+	ClassDB::bind_method(D_METHOD("standardize_locale", "locale", "add_defaults"), &TranslationServer::standardize_locale, DEFVAL(false));
```

Comment:
This will need compatibility method binding, see https://docs.godotengine.org/en/stable/contributing/development/handling_compatibility_breakages.html

Reply:
That was exactly what I was about to say. Thanks @bruvzg.